### PR TITLE
Initial Commit: Landing Page and Avatar Component

### PR DESCRIPTION
**Pull request by AshwiniSetty**
Added the home page and the Avatar component.

Please review my landing page and Avatar component from Docs

Link: https://miniwii-ui.netlify.app/

### CODE REVIEW

**Comment by k-rahul21** [comment]
code looks good to be merged.

File: components.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+
+.next-comp,.prev-comp{
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    transition: all .2s ease-in-out;
+}
+
+a{
```
Comment (sharmayatendra) [inline components.css:137]: try to use class instead of directly targeting the element

File: main.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    width: 500px;
+    margin: auto;
+    margin-bottom: 0.5rem;
+
+}
+
+/*main*/
+main .heading{
+    margin: auto;
+    max-width: 900px;
```
Comment (sharmayatendra) [inline main.css:20]: try to use relative units

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    display: inline;
+    padding: 0rem 0.5rem;
+}
+.footer ul{
+    padding-inline-start: 0px;
+}
+
+
+/*responsive code*/
+@media screen and (max-width : 1200px){
```
Comment (sharmayatendra) [inline styles.css:146]: here also use relative unit

**Review by sharmayatendra — APPROVED** [review]
can look upon suggested changes. good work

File: avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-light&wt=none&l=css&width=800&ds=true&dsyoff=100px&dsblur=100px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%252F*Image%2520circular%2520avatars%2520in%2520different%2520sizes*%252F%250A%253Cimg%2520class%253D%2522avatar%2520xlarge%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522xlarge%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520large%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522large%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520medium%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522medium%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520small%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522small%2520size%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520xsmall%2522%2520src%253D%2522%252Fimages%252Favatar1.jpg%2522%2520alt%253D%2522xsmall%2520size%2520avatar%2522%253E"
+                    style="height: 186px;"
```
Comment (bharati-21) [inline avatar/avatar.html:148]: Please avoid inline styles. You can create a separate class for carbon iframes, copy from here and paste it there.

File: avatar/avatar.html [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
```
Comment (bharati-21) [inline avatar/avatar.html:181]: Same comment here.

File: avatar/avatar.html [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
```
Comment (bharati-21) [inline avatar/avatar.html:215]: Please check here as well.

File: avatar/avatar.html [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-light&wt=none&l=css&width=800&ds=true&dsyoff=100px&dsblur=100px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2509%2509%252F*Square%2520Text%2520avatars%2520in%2520different%2520sizes*%252F%250A%2509%2509%253Cdiv%2520class%253D%2522avatar%2520square%2520avatar-text%2520xlarge%2522%2520role%253D%2522img%2522%253E%2520A%2520%253C%252Fdiv%253E%250A%2509%2509%253Cdiv%2520class%253D%2522avatar%2520square%2520avatar-text%2520large%2522%2520role%253D%2522img%2522%253E%2520A%2520%253C%252Fdiv%253E%250A%2509%2509%253Cdiv%2520class%253D%2522avatar%2520square%2520avatar-text%2520medium%2522%2520role%253D%2522img%2522%253E%2520A%2520%253C%252Fdiv%253E%250A%2509%2509%253Cdiv%2520class%253D%2522avatar%2520square%2520avatar-text%2520small%2522%2520role%253D%2522img%2522%253E%2520A%2520%253C%252Fdiv%253E%250A%2509%2509%253Cdiv%2520class%253D%2522avatar%2520square%2520avatar-text%2520xsmall%2522%2520role%253D%2522img%2522%253E%2520A%2520%253C%252Fdiv%253E"
+                    style="height: 186px;"
```
Comment (bharati-21) [inline avatar/avatar.html:243]: Here too.

File: main.css [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+.code-container::-webkit-scrollbar-thumb {
+    background-color: var(--light-gray-color);
+    
+}
+
+.code-container::-webkit-scrollbar-button {
+    margin: 0.5rem 0;
+}
+
+
```
Comment (bharati-21) [inline main.css:129]: Please do make sure you have consistent spacings between your code.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+    --success-hover: #008E89;
+
+    --active-hover : #084594;
+
+    --off-white : #F6F9FC;
+    --active-color : #1572A1;
+
+    --logo-fontsize: 1.4rem;
+    --nav-item-fontsize: 1rem;
+
```
Comment (bharati-21) [inline styles.css:37]: I saw most of the same color variables in component.css. You could look into this. Maybe remove it from there and sue from there. 👍🏽

**Review by bharati-21 — COMMENTED** [review]
Looks good to be merged. Only dropped a few comments for sanity checks. 👍🏽

**Pull request merged**